.env.example

```
@@ -0,0 +1,14 @@
+# Kings Roofing NC — Environment Variables
+# Copy this file to .env.local and fill in real values
+
+# Resend email (for contact form submissions)
+RESEND_API_KEY=re_xxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# Optional: PostHog analytics
+NEXT_PUBLIC_POSTHOG_API_KEY=
+NEXT_PUBLIC_POSTHOG_API_HOST=https://app.posthog.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_API_HOST key should go before the NEXT_PUBLIC_POSTHOG_API_KEY key

(UnorderedKey)